be/src/formats/parquet/schema.h

```diff
@@ -89,12 +89,29 @@ struct LevelInfo {
     std::string debug_string() const;
 };
 
+enum ColumnType { SCALAR = 0, ARRAY, MAP, STRUCT };
+
+inline std::string column_type_to_string(const ColumnType& column_type) {
```

Comment:
because this function is not necessary to be inlined for performance. It will be good for compile performance if you move it into cpp files.

Reply:
done